Correct package name of fragment

App crashes with error:
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.haris.houlis.moviemaze/com.haris.houlis.moviemaze.ui.MainActivity}: android.view.InflateException: Binary XML file line #2 in com.haris.houlis.moviemaze:layout/activity_main: Binary XML file line #2 in com.haris.houlis.moviemaze:layout/activity_main: Error inflating class androidx.fragment.app.FragmentContainerView
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3270)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
     Caused by: android.view.InflateException: Binary XML file line #2 in com.haris.houlis.moviemaze:layout/activity_main: Binary XML file line #2 in com.haris.houlis.moviemaze:layout/activity_main: Error inflating class androidx.fragment.app.FragmentContainerView
     Caused by: android.view.InflateException: Binary XML file line #2 in com.haris.houlis.moviemaze:layout/activity_main: Error inflating class androidx.fragment.app.FragmentContainerView
     Caused by: androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment com.haris.houlis.movienaze.ui.browseMovies.MoviesFragment: make sure class name exists
```

--- a/app/src/main/res/layout/activity_main.xml
+++ b/app/src/main/res/layout/activity_main.xml
@@ -2,7 +2,7 @@
 <androidx.fragment.app.FragmentContainerView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/main_activity_root"
-    android:name="com.haris.houlis.movienaze.ui.browseMovies.MoviesFragment"
+    android:name="com.haris.houlis.moviemaze.ui.browseMovies.MoviesFragment"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".ui.MainActivity" />

